examples/algorithms/plot_betweenness_centrality.py

```diff
@@ -0,0 +1,74 @@
+"""
+=====================
+Betweeness Centrality
+=====================
+
+Betweenness centrality measures of positive gene functional associations
+using WormNet v.3-GS.
+
+Data from: https://www.inetbio.org/wormnet/downloadnetwork.php
+"""
+
+import networkx as nx
+import matplotlib.pyplot as plt
+
+# Gold standard data of positive gene functional associations
+# from https://www.inetbio.org/wormnet/downloadnetwork.php
+G = nx.read_edgelist("WormNet.v3.benchmark.txt")
+
+# largest connected component
+components = nx.connected_components(G)
+largest_component = max(components, key=len)
+H = G.subgraph(largest_component)
+
+# compute centrality
+centrality = nx.betweenness_centrality(H, k=10, endpoints=True)
+
+#### draw graph ####
+fig, ax = plt.subplots(figsize=(20, 15))
+pos = nx.spring_layout(H, k=0.15, seed=4552321)
+node_color = [d * 10 for n, d in H.degree()]
```

Comment:
You could try coloring by community structure, with size by betweenness.
It takes a couple minutes on my machine to compute the community. 
But the node colors nicely show the communities that are already obvious from positions using spring_layout.

```suggestion
# compute community structure
lpc = nx.community.label_propagation_communities(H)
community_index = {n: i for i, com in enumerate(lpc) for n in com}

#### draw graph ####
fig, ax = plt.subplots(figsize=(20, 15))
pos = nx.spring_layout(H, k=0.15, seed=4552321)
node_color = [community_index[n] for n in H]
```